boards/common/iotlab/include/board_common.h

```diff
@@ -49,46 +49,46 @@ extern "C" {
  * Tell the xtimer that we use a 16-bit peripheral timer
  * @{
  */
-#define XTIMER_WIDTH        (16U)
-#define XTIMER_OVERHEAD     (6U)
-#define XTIMER_SHOOT_EARLY  (3U)
+#define XTIMER_WIDTH              (16)
```

Comment:
changing `16U` to `16` should not be part of this PR...

Reply:
It was requested by @smlng [some times ago](https://github.com/RIOT-OS/RIOT/pull/7937#discussion_r162161669), I made that consistent across other board configurations.